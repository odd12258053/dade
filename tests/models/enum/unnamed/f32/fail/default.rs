use dade::model;
#[model]
enum TestModel {
    Value(
       #[field(default = 1)]
        f32
    ),
}
fn main() {}
