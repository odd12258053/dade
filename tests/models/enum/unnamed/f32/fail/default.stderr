error[E0308]: `?` operator has incompatible types
 --> tests/models/enum/unnamed/f32/fail/default.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^ expected `f32`, found integer
  |
  = note: `?` operator cannot convert from `{integer}` to `f32`
  = note: this error originates in the attribute macro `model` (in Nightly builds, run with -Z macro-backtrace for more info)
